### PR TITLE
acid spray now deletes razorwire again

### PR DESCRIPTION
**Pull request by MacBlaze1**
## About The Pull Request

acid spray now deletes razorwire again

## Why It's Good For The Game
lumi said so

## Changelog

:cl:

fix: fixed acid spray not deleting razorwire

/:cl:

### CODE REVIEW

**Comment by Strayed-Raven** [comment]
>lumi said so
Wew, Fix bug. Bug good?

File: code/modules/mob/living/carbon/xenomorph/xenoprocs.dm [hunk 1]
```diff
@@ -404,7 +404,7 @@
 	return TRUE // normal density flag
 
 /obj/structure/razorwire/acid_spray_act(mob/living/carbon/xenomorph/X)
-	. = ..()
+	take_damage(100, "acid", "acid")
```
Comment (Lumipharon) [inline code/modules/mob/living/carbon/xenomorph/xenoprocs.dm:407]: use defines.

Comment (Lumipharon) [reply code/modules/mob/living/carbon/xenomorph/xenoprocs.dm:407]: Also the damage should be what it was, which is double acid_spray_structure_damage. Not all xenos could actually insta break razor.

**Review by Lumipharon — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/xenomorph/xenoprocs.dm (hunk 1 above)
Comment (MacBlaze1) [reply code/modules/mob/living/carbon/xenomorph/xenoprocs.dm]: fixed

**Review by Lumipharon — APPROVED** [review]

**Pull request merged**